### ISSUE TITLE
Why we have to build tensorflow from source?

### ISSUE THREAD

**Issue by GeKeShi** [wenwei202/terngrad#13]
Can we install the official TensorFlow binary version and run your code based on it?  Does that mean you changed the source code of TensorFlow? I find that your algorithm just inserts some tensor operation in the computation graph, but have not changed the source code of TensorFlow.

**Comment by woskii** [comment]
I have this question too. Have you figured out the necessary to build from source?

**Comment by GeKeShi** [comment]
no, I haven't, the author implements the code in  official Pytorch code, it means it will make no sense to  build the tensorflow version.

**Comment by GeKeShi** [comment]
> I have this question too. Have you figured out the necessary to build from source?

no, I haven't, the author implements the code in official Pytorch code, it means it will make no sense to build the tensorflow version.

**Comment by woskii** [comment]
> > I have this question too. Have you figured out the necessary to build from source?
> 
> no, I haven't, the author implements the code in official Pytorch code, it means it will make no sense to build the tensorflow version.

Can you post a link of the pytorch implementation? Thanks a lot.

**Comment by GeKeShi** [comment]
> > > I have this question too. Have you figured out the necessary to build from source?
> > 
> > 
> > no, I haven't, the author implements the code in official Pytorch code, it means it will make no sense to build the tensorflow version.
> 
> Can you post a link of the pytorch implementation? Thanks a lot.

The author post that link in the README of this repo
